openapi.yaml

```diff
@@ -80,6 +80,7 @@ paths:
       tags:
         - Messages
       x-sdkOperation: create
+      x-isPost: true
```

Comment:
Workaround for Whispir not being strictly RESTful - we can use this to conditionally parse the `Location` header for POST endpoints.